rasa/shared/core/domain.py

```diff
@@ -1219,19 +1228,17 @@ def slots_for_entities(self, entities: List[Dict[Text, Any]]) -> List[SlotSet]:
         if self.store_entities_as_slots:
             slot_events = []
             for slot in self.slots:
-                if slot.auto_fill:
-                    matching_entities = [
-                        entity.get("value")
-                        for entity in entities
-                        if entity.get("entity") == slot.name
-                    ]
-                    if matching_entities:
-                        if slot.type_name == "list":
-                            slot_events.append(SlotSet(slot.name, matching_entities))
-                        else:
-                            slot_events.append(
-                                SlotSet(slot.name, matching_entities[-1])
-                            )
+                # if slot.auto_fill:
```

Comment:
This was a temp fix to prevent quite a few tests from failing since this method is used in several other parts of the codebase. I will revisit the usage of this method once the 2nd issue of the implementation is completed too to see if this can be adapted/removed.

Reply:
This means we are always auto-filling slots from entities though right? Shouldn't this just be entirely replaced by the mappings now?

Reply:
>  This means we are always auto-filling slots from entities though right?

Only when the entity and slot names match.

>  Shouldn't this just be entirely replaced by the mappings now?

Oh right, you mean modify the implementation of this method to find the matching mapping (i.e, via `SlotMapping.entity_is_desired`?